changelogs/fragments/fix-fetch-return-file.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - fetch - also return the source file name in the `file` attribute on file change
```

Per https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment, how about something like:

```suggestion
  - fetch - also return ``file`` in the result when changed is ``True`` (https://github.com/ansible/ansible/pull/85729).
```

---

Thanks. Done.